.github/workflows/build.yaml

```diff
@@ -34,22 +35,18 @@ jobs:
       - name: Run lint check
         if: ${{ matrix.python-version == '3.11' }}
         run: |
-          venv/bin/ruff format --check obs_common tests
-          venv/bin/ruff check obs_common tests
+          PATH="venv/bin:$PATH" bin/lint.sh
       - name: Run tests
         env:
           SENTRY_DSN: http://public@localhost:8090/1
           STORAGE_EMULATOR_HOST: http://localhost:8001
+          PUBSUB_EMULATOR_HOST: localhost:5010
         run: |
-          docker compose up -d fakesentry gcs-emulator
-          venv/bin/pytest tests/
+          docker compose up -d fakesentry gcs-emulator pubsub
+          # Run outside docker because we are testing the matrix python version
+          PATH="venv/bin:$PATH" bin/test.sh
```

Comment:
Ah. At first I was wondering "what is The Matrix TM", but I see we are running the tests against multiple versions of Python: https://github.com/mozilla-services/obs-common/blob/4a78aadca50bdf890ebd85a8b0e7993d313a7f08/.github/workflows/build.yaml#L17.